sql/moz-fx-data-shared-prod/static/country_codes_v1/data.csv

```diff
@@ -1,253 +1,253 @@
-"Taiwan, Province of China",TW,TWN,Asia,Eastern Asia,Unspecified,FALSE,FALSE,FALSE
-Tajikistan,TJ,TJK,Asia,Central Asia,Unspecified,FALSE,FALSE,FALSE
-"Tanzania, United Republic of",TZ,TZA,Africa,Sub-Saharan Africa,Eastern Africa,FALSE,FALSE,FALSE
-Thailand,TH,THA,Asia,South-eastern Asia,Unspecified,FALSE,FALSE,FALSE
-Timor-Leste,TL,TLS,Asia,South-eastern Asia,Unspecified,FALSE,FALSE,FALSE
-Togo,TG,TGO,Africa,Sub-Saharan Africa,Western Africa,FALSE,FALSE,FALSE
-Tokelau,TK,TKL,Oceania,Polynesia,Unspecified,FALSE,FALSE,FALSE
-Tonga,TO,TON,Oceania,Polynesia,Unspecified,FALSE,FALSE,FALSE
-Trinidad and Tobago,TT,TTO,Americas,Latin America and the Caribbean,Caribbean,FALSE,FALSE,FALSE
-Tunisia,TN,TUN,Africa,Northern Africa,Unspecified,FALSE,FALSE,FALSE
-Turkey,TR,TUR,Asia,Western Asia,Unspecified,FALSE,FALSE,FALSE
-Turkmenistan,TM,TKM,Asia,Central Asia,Unspecified,FALSE,FALSE,FALSE
-Turks and Caicos Islands,TC,TCA,Americas,Latin America and the Caribbean,Caribbean,FALSE,FALSE,FALSE
-Tuvalu,TV,TUV,Oceania,Polynesia,Unspecified,FALSE,FALSE,FALSE
-Uganda,UG,UGA,Africa,Sub-Saharan Africa,Eastern Africa,FALSE,FALSE,FALSE
-Ukraine,UA,UKR,Europe,Eastern Europe,Unspecified,FALSE,FALSE,FALSE
-United Arab Emirates,AE,ARE,Asia,Western Asia,Unspecified,FALSE,FALSE,FALSE
-United Kingdom,GB,GBR,Europe,Northern Europe,Unspecified,TRUE,TRUE,TRUE
-United States,US,USA,Americas,Northern America,Unspecified,TRUE,TRUE,TRUE
-United States Minor Outlying Islands,UM,UMI,Oceania,Micronesia,Unspecified,FALSE,FALSE,FALSE
-Uruguay,UY,URY,Americas,Latin America and the Caribbean,South America,FALSE,FALSE,FALSE
-Uzbekistan,UZ,UZB,Asia,Central Asia,Unspecified,FALSE,FALSE,FALSE
-Vanuatu,VU,VUT,Oceania,Melanesia,Unspecified,FALSE,FALSE,FALSE
-"Venezuela, Bolivarian Republic of",VE,VEN,Americas,Latin America and the Caribbean,South America,FALSE,FALSE,FALSE
-Viet Nam,VN,VNM,Asia,South-eastern Asia,Unspecified,FALSE,FALSE,FALSE
-"Virgin Islands, British",VG,VGB,Americas,Latin America and the Caribbean,Caribbean,FALSE,FALSE,FALSE
-"Virgin Islands, U.S.",VI,VIR,Americas,Latin America and the Caribbean,Caribbean,FALSE,FALSE,FALSE
-Wallis and Futuna,WF,WLF,Oceania,Polynesia,Unspecified,FALSE,FALSE,FALSE
-Western Sahara,EH,ESH,Africa,Northern Africa,Unspecified,FALSE,FALSE,FALSE
-Yemen,YE,YEM,Asia,Western Asia,Unspecified,FALSE,FALSE,FALSE
-Zambia,ZM,ZMB,Africa,Sub-Saharan Africa,Eastern Africa,FALSE,FALSE,FALSE
-Zimbabwe,ZW,ZWE,Africa,Sub-Saharan Africa,Eastern Africa,FALSE,FALSE,FALSE
-Kosovo,XK,XKK,Unspecified,Unspecified,Unspecified,FALSE,FALSE,FALSE
-Rest of World,ROW,ROW,Unspecified,Unspecified,Unspecified,FALSE,FALSE,FALSE
-Unknown Country Name,??,???,Unknown Region,Unknown Sub-region,Unknown Intermediate Region,FALSE,FALSE,FALSE
+name,code,code_3,region_name,subregion_name,intermediate_region_name,pocket_available_on_newtab,mozilla_vpn_available,sponsored_tiles_available_on_newtab,ads_value_tier
+Afghanistan,AF,AFG,Asia,Southern Asia,Unspecified,FALSE,FALSE,FALSE,null
```

Comment:
@sergiosonline this is resulting in the literal string "null" ending up in the `ads_value_tier` column.  If you want actual nulls in that column then its mode would need to be changed from `REQUIRED` to `NULLABLE` and the "null" values in this file should be removed (but not the commas, basically leaving empty values for the column).